tests/test_dm_lab.py

```diff
@@ -74,8 +75,8 @@ def test_seeding(level_name):
     env_2.close()
 
 
-@pytest.mark.skip(reason="DM lab tests are not currently passing and are disabled.")
-@pytest.mark.parametrize("level_name", LEVEL_NAMES)
+@pytest.mark.skip(reason="Seeding tests are not currently possible for DM Lab.")
+@pytest.mark.parametrize("level_name", PASSING_LEVEL_NAMES)
```

Comment:
Why are these skipped then parameterized?

Reply:
Good point I'll leave them to all envs